templates/Sparkle/header.tpl

```diff
@@ -63,7 +63,7 @@
 					<if \Froxlor\Settings::Get('panel.allow_theme_change_customer') == '1' && $userinfo['adminsession'] == 0>
 						<li><a href="{$linker->getLink(array('section' => 'index', 'page' => 'change_theme'))}">{$lng['panel']['theme']}</a></li>
 					</if>
-					<if \Froxlor\Settings::Get('api.enabled') == 1>
+					<if \Froxlor\Settings::Get('api.enabled') == 1 && $userinfo['api_allowed'] == 1>
```

Comment:
For me a quick hack for preventing customers asking for / about the api. I disabled api access for the customers from the beginning because most of them has no clue.

Reply:
Well, mostly that is what you want: ah you want api acces (because I have this enabled for the system)? okay, pay something extra (for example) - of course there is the other side: if I've disabled api for a customer I do not want him to see anything of it. It's not an easy decision :)

Reply:
I can't give you the right answer. Maybe a checkbox in the adminsection "Show api commands if not enabled?"

Reply:
Why not just both:
```php
\Froxlor\Settings::Get('api.enabled') == 1 && ($userinfo['api_allowed'] == 1 || $userinfo['api_allowed'] == 0 && $user_still_has_api_keys )
```
Of course `$user_still_has_api_keys` needs to be implemented